packages/@lightningjs/ui-components/src/components/Tile/Tile.test.js

```diff
@@ -433,19 +433,16 @@ describe('Tile', () => {
     expect(tile._ProgressBar).toBeUndefined();
   });
 
-  it('should update metadata and remove if no longer needed if metadataLocation is inset', async () => {
+  it('should update metadata and remove if no longer needed', async () => {
     expect(tile.metadata).toBeUndefined();
+
     tile.metadata = { title: 'test' };
-    tile.metadataLocation = 'inset';
     await tile.__updateMetadataSpyPromise;
     expect(tile.metadata).not.toBeUndefined();
+
     tile.metadata = undefined;
     await tile.__updateMetadataSpyPromise;
     expect(tile._Metadata).toBeUndefined();
```

Comment:
I think adding the opposite of this assertion before setting `tile.metadata = undefined;` would be good here so that we're certain `tile._Metadata` was defined at the start of the test and was only removed after `tile.metadata` was set to `undefined`.
```js
expect(tile._Metadata).not.toBeUndefined()
```